theme/templates/base.html

```diff
@@ -4,6 +4,7 @@
   {% block head %}
   <title>{% block title %}{{ SITENAME }}{% endblock title %}</title>
   <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
oh my, we were missing that? No wonder scrolling on iOS is funky. 🎉

Reply:
Yes, took me some time to find it !